tests/specs/3way/breakoutRooms.spec.ts

```diff
@@ -9,6 +9,8 @@ const LIST_ITEM_CONTAINER = 'list-item-container';
 
 describe('BreakoutRooms', () => {
     it('check support', async () => {
+        await hangupAllParticipants();
+
```

Comment:
Why have you add this? It is the beginning of the tests and there is no need to hangup anything?

Reply:
This one is on another commit. I noticed that those randomly fail for me when I start all the tests and when I start only this test it never fails. So I figured that it must be some leftovers from the previous test that are making it fail. Am I wrong?

Do we already hangup before the test?

Reply:
yeah. wdio prepares new browser sessions for us in the beginning of new test.suite. So the first `it` is clean :)

Reply:
https://github.com/jitsi/jitsi-meet/blob/a6d333c07ae80b5e79a28bdd50521f1374bad577/tests/wdio.conf.ts#L175

This is the time when we setup the logger and initialize ctx.